pandas/tests/indexing/test_timedelta.py

```diff
@@ -13,7 +13,24 @@ def test_boolean_indexing(self):
                          [0, 1, 2, 10, 4, 5, 6, 7, 8, 9],
                          [10, 10, 10, 3, 4, 5, 6, 7, 8, 9]]
         for cond, data in zip(conditions, expected_data):
-            result = df.assign(x=df.mask(cond, 10).astype('int64'))
+            result = df.assign(x=df.mask(cond, 10).astype(df['x'].dtype))
+            expected = pd.DataFrame(data,
+                                    index=pd.to_timedelta(range(10), unit='s'),
+                                    columns=['x'])
+            tm.assert_frame_equal(expected, result)
+
+    def test_slice_indexing(self):
```

Comment:
Hi, in the new case we are not passing a boolean mask. Would you agree if I renamed the test to `test_list_like_indexing`?